### DISCUSSION TITLE
Inshackle Instagram  not getting banned

### DISCUSSION THREAD

**Discussion by viprdiamondent** [cyberkallan/inshackle-bot#24] [Ideas]
Evening Cyber Kallan my name is Aaron and I really like your Instagram Bot,  Your Instagram Bot  works really good I was gaining a lot of followers,  only thing is, the people from instagram bann accounts using this Repository
Is there any way we could use your Repository without getting banned on instagram?

**Comment by pxraxgit** [comment]
what is password /
